engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-traces-modal.html

```diff
@@ -0,0 +1,75 @@
+<div data-extend-template="components/views/bst-modal.html">
+  <h4 data-block="modal-header">Content Host Tracer Management</h4>
+
+  <div data-block="modal-body">
+    <form id="traceActionForm" name="traceActionForm" class="form" method="post" action="/katello/remote_execution">
+      <input type="hidden" name="remote_action" value="service_restart"/>
+      <input type="hidden" name="name" ng-value="traceActionFormValues.service"/>
+      <input type="hidden" name="host_ids" ng-value="traceActionFormValues.hostIds"/>
+      <input type="hidden" name="customize" ng-value="traceActionFormValues.customize"/>
+      <input type="hidden" name="authenticity_token" ng-value="traceActionFormValues.authenticityToken"/>
+    </form>
+
+    <div class="row">
+      <div class="col-sm-12">
+        <div bst-global-notification></div>
+      </div>
+    </div>
+
+    <div data-extend-template="layouts/partials/table.html">
+      <div data-block="search" ng-show=tracesearch></div>
+      <span data-block="list-actions" ng-hide="contentHost.readonly">
+        <div bst-modal="performViaRemoteExecution(true)">
+          <div data-block="modal-header" translate>Restart services on the selected Content Hosts ?</div>
+          <div data-block="modal-body" translate>Are you sure you want to restart the services on the content hosts chosen?</div>
+          <span data-block="modal-confirm-button">
+            <button class="btn btn-danger" ng-click="ok()">
+              <span translate>Restart</span>
+            </button>
+          </span>
+        </div>
+  
+          <span class="btn-group">
+            <button class="btn btn-default" type="button"
+                    translate
+                    ng-hide="denied('edit_hosts', host)"
+                    ng-disabled="table.getSelected().length == 0 || !remoteExecutionPresent"
+                    ng-click="openModal()">
+              Restart Selected
+            </button>
+          </span>
+      </span>
+  
+      <span data-block="no-rows-message" translate>
+        There are no Traces to display.
+      </span>
+  
+      <div data-block="table">
+        <div data-extend-template="layouts/select-all-results.html"></div>
+        <table bst-table="table" class="table table-striped table-bordered" ng-class="{'table-mask': table.working}">
+          <thead>
+            <tr bst-table-head row-select>
+              <th bst-table-column="application" translate>Application</th>
+              <th bst-table-column="app_type" translate>Type</th>
+              <th bst-table-column="helper" translate>Helper</th>
+              <th bst-table-column="affected_hosts" translate>Hostname</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr bst-table-row ng-repeat="trace in table.rows | filter:table.tracesCompare" row-select="trace">
```

Comment:
I'm not seeing a `tracesCompare` function in this PR - what does it do for a regular content host?

Reply:
In the single content host traces tab there is a search box, I will remove it from here, left over copy and pasta while getting the structure built. Will remove.